src/content/get-started/ch2-add-commands.md

```diff
@@ -0,0 +1,514 @@
+---
+title: Making Your CLI Program Interactive
+description: Add simple commands to your cli program. Learn the fundamentals of Dart syntax including control flow, collections, variables, functions, and more.
+---
+
+In this chapter, you'll get hands-on with Dart syntax. You'll cover control
+flow, collections, variables, functions, and more. You'll learn how to read user
+input, print usage information, and create a basic command-line interaction. By
+the end, you'll have a foundational understanding of Dart syntax.
+
+## Prerequisites
+
+* Completed Chapter 1 and have a working Dart development environment.
+* Familiarity with basic programming concepts (variables, data types, control
+  low).
+
+## Tasks
+
+Start building your Wikipedia command-line application by adding some basic
+functionality and exploring Dart syntax.
```

Comment:
Again, I'd name this example something else. Even "Dictionary" is better, but I'm sure you can come up with something even better. :)

Reply:
I'm working with dartpedia for now, unless our super official chat-room vote decides otherwise.